### PR TITLE
Aodn wave dm add dta

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

File: workspace/AODN_WAVE_DM/process/DataHarvest/DataHarvest_0.1.item [hunk 1]
```diff
@@ -178,137 +241,123 @@
...
+    <elementParameter field="CLOSED_LIST" name="DB_VERSION" value="PRIOR_TO_V9"/>
     <elementParameter field="TEXT" name="HOST" value="context.Destination_Server"/>
     <elementParameter field="TEXT" name="PORT" value="context.Destination_Port"/>
     <elementParameter field="TEXT" name="DBNAME" value="context.Destination_Database"/>
     <elementParameter field="TEXT" name="SCHEMA_DB" value="context.Destination_Schema"/>
-    <elementParameter field="TEXT" name="TYPE" value="POSTGRESQL"/>
+    <elementParameter field="TEXT" name="TYPE" value="POSTGRESQL" show="false"/>
     <elementParameter field="TEXT" name="USER" value="context.Destination_Login"/>
-    <elementParameter field="TEXT" name="PASS" value="context.Destination_Password"/>
+    <elementParameter field="PASSWORD" name="PASS" value="8pW6Nmy2kuc8D4KM9GKIHFjfbxE3isCHr24cFevwCvw="/>
```
Comment (bpasquer) [inline workspace/AODN_WAVE_DM/process/DataHarvest/DataHarvest_0.1.item]: you probably want to change that back to the context value

Comment (lbesnard) [reply workspace/AODN_WAVE_DM/process/DataHarvest/DataHarvest_0.1.item]: I have the feeling talend is hashing the context.Destination_Password name (not  the value). Could you please confirm @jonescc ?

Comment (lbesnard) [reply workspace/AODN_WAVE_DM/process/DataHarvest/DataHarvest_0.1.item]: @bpasquer just did a fing (ag) on the whole repo
```
ag "<elementParameter field=\"PASSWORD\" name=\"PASS\" value="
```


and found 

```
ARGO/process/ArgoHarvester/Subjobs/Profiles/HarvestArgoProfile_0.1.item
102:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/>
609:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/>
701:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/>
954:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/>


SOOP_TRV/process/SOOP_TRV/SOOP_TRV_harvester_0.1.item                                                                                                                                         
114:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/> 
180:    <elementParameter field="PASSWORD" name="PASS" value="8pW6Nmy2kuc8D4KM9GKIHFjfbxE3isCHr24cFevwCvw="/>
217:    <elementParameter field="PASSWORD" name="PASS" value="8pW6Nmy2kuc8D4KM9GKIHFjfbxE3isCHr24cFevwCvw="/>
319:    <elementParameter field="PASSWORD" name="PASS" value="8pW6Nmy2kuc8D4KM9GKIHFjfbxE3isCHr24cFevwCvw="/>
                                                                                               
SRS_SURFACE_WAVES/process/SRS_SURFACE_WAVES_harvester_1.0.item                                                                                                                                
114:    <elementParameter field="PASSWORD" name="PASS" value="0RMsyjmybrE=" show="false"/>     
                                                                                                           

```

So I reckon it's all good

Comment (bpasquer) [reply workspace/AODN_WAVE_DM/process/DataHarvest/DataHarvest_0.1.item]: 👍

**Pull request merged**